### PR TITLE
Generate tasks on the worker

### PR DESCRIPTION
**Pull request by dereklieu**

### CODE REVIEW

File: scripts/to-geojson.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ merge(ways, tags, toKey)
   next(null, {
     type: 'Feature',
     properties: properties,
+    _id: wayArr[0].way_id,
```
Comment (mileswwatkins) [inline scripts/to-geojson.js]: Looks good; can we call this `way_id` instead?

**Review by mileswwatkins — APPROVED** [review]

**Comment by matthewhanson** [comment]
I made the change to way_id @mileswwatkins @dereklieu so I could get this merged to master so I can get a new image pushed for deployment. Any reason to go back to _id ?

**Comment by mileswwatkins** [comment]
@matthewhanson, I'd assumed that @dereklieu was referring to `_id` somewhere in his front-end code, but not sure.

**Comment by dereklieu** [comment]
This is referenced elsewhere but I can fix those later.

**Pull request merged**